optimize_sidebars.js

```diff
@@ -793,9 +793,14 @@ module.exports = {
                 "components/userguide/view-mode",
               ],
             },
+            {
+              "Dashboards maintained by Camunda": [
```

Comment:
What does this mean?

Reply:
See https://camunda.slack.com/archives/C04E567BZTQ/p1679670215084769?thread_ts=1679667868.167339&cid=C04E567BZTQ

Reply:
Well, the management dashboard is also maintained by Camunda, and in that page the user would always be clicking to open an instant preview dashboard. To me it makes sense, what do you think @toco-cam ?